### ISSUE TITLE
Как работает RANDOM_CLICKS_COUNT?

### ISSUE THREAD

**Issue by bd100b** [shamhi/HamsterKombatBot#115]
Правильно ли я понимаю что RANDOM_CLICKS_COUNT если выставлено например [50,200] то оно за одну секунду может послать например 150 кликов?

**Comment by XxHLeB** [comment]
> Правильно ли я понимаю что RANDOM_CLICKS_COUNT если выставлено например [50,200] то оно за одну секунду может послать например 150 кликов?

Да, это идет минимальное значение и максимальное, попадается в промежутке между этими двумя числами

**Comment by ButCheR-PRO** [comment]
Есть изменения. В файле .env:
```
SLEEP_BY_MIN_ENERGY=[1800,2400]
RANDOM_TAPS_COUNT=[50,200]
SLEEP_BETWEEN_TAP=[10,25]
```
